docs/Game/Unreal/getting-started/01-connect-unreal.md

```diff
@@ -5,40 +5,88 @@ id: unreal-connect-wallet
 
 This section assumes you have already deployed the relevant smart contracts to the blockchain and have smart contract addresses and ABI.
 
+#### STEP ONE
 
-1. Upon initialization, the client is called and a `device_id` is created.  
+Upon initialization, a unique `deviceId` is generated.
 
-	```js
-	deviceId = load->UniqueId;
-	baseUrl = "http://45.77.189.28:5000/";
-	```
+```js
+deviceId = load->UniqueId;
+baseUrl = "http://45.77.189.28:5000/";
+```
 
-2. The Connection status is queried and a `sessionId`is assigned subject to the Login Status.
+#### STEP TWO
 
-	```js
-	GetClient(FMirageConnectionStatus Status)
-	```
+Use the **GetClient** method to send a request to http://45.77.189.28:5000/connect. 
```

Comment:
Shouldn't a method be formatted as code? It is part of code.

Maybe `GetClient`?